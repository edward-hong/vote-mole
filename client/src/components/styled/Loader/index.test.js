import React from 'react'
import { render } from '@testing-library/react'

import Loader from '.'

describe('Loader', () => {
	it('renders correctly', () => {
		const {
			container: { firstChild: element },
		} = render(<Loader />)

		expect(element).toMatchSnapshot()
	})
})
