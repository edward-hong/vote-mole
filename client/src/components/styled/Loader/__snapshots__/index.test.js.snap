// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loader renders correctly 1`] = `
.c0 {
  position: relative;
  margin-top: 30px;
}

.c1 {
  margin: -24px 0 0 -24px;
  height: 49px;
  width: 49px;
  -webkit-animation: cssload-rotate 1150ms linear infinite;
  animation: cssload-rotate 1150ms linear infinite;
}

.c1,
.c1:before,
.c1:after {
  position: absolute;
  top: 50%;
  left: 50%;
  border: 1px solid #ccc;
  border-left-color: black;
  border-radius: 974px;
}

.c1:before {
  content: '';
  margin: -22px 0 0 -22px;
  height: 43px;
  width: 43px;
  -webkit-animation: cssload-rotate 1150ms linear infinite;
  animation: cssload-rotate 1150ms linear infinite;
}

.c1:after {
  content: '';
  margin: -28px 0 0 -28px;
  height: 55px;
  width: 55px;
  -webkit-animation: cssload-rotate 2300ms linear infinite;
  animation: cssload-rotate 2300ms linear infinite;
}

<div
  class="c0"
>
  <div
    class="c1"
  />
</div>
`;
